pkgs/top-level/python-packages.nix

```diff
@@ -4688,9 +4688,14 @@ in {
   else
     callPackage ../development/python-modules/pillow {
       inherit (pkgs) freetype libjpeg zlib libtiff libwebp tcl lcms2 tk;
-      inherit (pkgs.xorg) libX11;
+      inherit (pkgs.xorg) libX11 libxcb;
     };
 
+  pillow-simd = callPackage ../development/python-modules/pillow-simd {
```

Comment:
cc @FRidh can this not cause conflicts?

Reply:
They might be propagated into the same PYTHONPATH and one would shadow the other. I think we also have build time checks to detect this. However we have something similar for tensorflow afaik.

Reply:
I couldn't find anything that does that. Yeah they can end up in the same PYTHONPATH but people just need to be careful.